labs-ios-starter/Cities/CitiesViewController.swift

```diff
@@ -30,10 +35,8 @@ extension CitiesViewController: UICollectionViewDelegate, UICollectionViewDataSo
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "cityCell", for: indexPath) as! CityCollectionViewCell
         
         let city: String = citySearchResults[indexPath.row]
-        let cityName: String = String(city.dropLast(3))
-        let state: String = String(city.dropFirst(city.count-2))
 
-        cell.cityStateLabel.text = "\(cityName), \(state)"
+        cell.cityStateLabel.text = city
         cell.populationLabel.text = "Population: 2.7M"
```

Comment:
Maybe we should get rid of these for now and add then when and if we have the other data.

Reply:
I think i'll remove the populationLabel altogether, good catch